clients/lambda.d.ts

```diff
@@ -254,11 +254,11 @@ declare class Lambda extends Service {
    */
   invoke(callback?: (err: AWSError, data: Lambda.Types.InvocationResponse) => void): Request<Lambda.Types.InvocationResponse, AWSError>;
   /**
-   *  For asynchronous function invocation, use Invoke.  Invokes a function asynchronously.
```

Comment:
Possible other improvements:

- [ ] Remove double space `  ` between the sentences
- [ ] Swap the order, so we describe first what this function does and then the synchronous alternative, which would make more sense semantically.